spark-commons/src/main/scala/za/co/absa/spark/commons/implicits/DataFrameImplicits.scala

```diff
@@ -49,21 +48,45 @@ object DataFrameImplicits {
       gatherData(showFnc)
     }
 
+    /**
+     * Get the string representation of the data in the format as [[org.apache.spark.sql.Dataset#show(truncate:Boolean):Unit* spark.Dataset.show(truncate: Boolean)]] displays them
+     *
+     * @return  The string representation of the data in the DataFrame
+     * @since 0.2.0
+     */
     def dataAsString(truncate: Boolean): String = {
       val showFnc:  () => Unit = ()=>{df.show(truncate)}
       gatherData(showFnc)
     }
 
+    /**
+     * Get the string representation of the data in the format as [[org.apache.spark.sql.Dataset#show(numRows:Int,truncate:Boolean):Unit* spark.Dataset.show(numRows: Int, truncate: Boolean)]] displays them
+     *
+     * @return  The string representation of the data in the DataFrame
+     * @since 0.2.0
+     */
     def dataAsString(numRows: Int, truncate: Boolean): String = {
       val showFnc: ()=>Unit = () => df.show(numRows, truncate)
       gatherData(showFnc)
     }
 
+    /**
+     * Get the string representation of the data in the format as [[org.apache.spark.sql.Dataset#show(numRows:Int,truncate:Int):Unit* spark.Dataset.show(numRows: Int, truncate: Int)]] displays them
```

Comment:
Why all those `#` there? I couldn't find any reference to it in https://github.com/ThoughtWorksInc/sbt-api-mappings, and it seems to be working fine with `.` - including 'click to go to the definition' in the IntelliJ and `sbt doc` also seems to be producing no warnings

Reply:
It even has Spark Mapping: https://github.com/ThoughtWorksInc/sbt-api-mappings/blob/5b1e4f11647421d287d3a26ed7fb077474ea7a3e/src/main/scala/com/thoughtworks/sbtApiMappings/SparkApiMappingRule.scala#L22

Reply:
All my magic power comes from this page and one longer method header. ([See](https://github.com/scala/scala/blob/2.12.x/test/scaladoc/resources/links.scala))

Reply:
Hmm okay, it's just a code docs, but it is from Scala 🤔 

well, I still think that we should use `.` primarily if it works: a) the destination is clickable and b) the `sbt doc` would be okay with it. And I think that many of these `#` can be replaced by `.` based on what I tried. I'll leave it up to you though